cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js

```diff
@@ -0,0 +1,260 @@
+    description: 'Created by autotest team',
+  };
+  const requestPolicyBody = {
+    requestTypes: ['Page'],
+    name: `hold${getRandomPostfix()}`,
+    id: uuid(),
+  };
+
+  before('Preconditions', () => {
+    cy.getAdminToken()
+      .then(() => {
+        ServicePoints.createViaApi(testData.userServicePoint);
+        testData.defaultLocation = Location.getDefaultLocation(testData.userServicePoint.id);
+        Location.createViaApi(testData.defaultLocation);
+        cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => {
+          testData.instanceTypeId = instanceTypes[0].id;
+        });
+        cy.getHoldingTypes({ limit: 1 }).then((holdingTypes) => {
+          testData.holdingTypeId = holdingTypes[0].id;
+        });
+        cy.createLoanType({
+          name: `type_${getRandomPostfix()}`,
+        }).then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
+          testData.materialTypeId = materialTypes.id;
+        });
+      })
+      .then(() => {
+        InventoryInstances.createFolioInstanceViaApi({
+          instance: {
+            instanceTypeId: testData.instanceTypeId,
+            title: itemData.title,
+          },
+          holdings: [
+            {
+              holdingsTypeId: testData.holdingTypeId,
+              permanentLocationId: testData.defaultLocation.id,
+            },
+          ],
+          items: [
+            {
+              barcode: itemData.barcode,
+              status: { name: 'Available' },
+              permanentLoanType: { id: testData.loanTypeId },
+              materialType: { id: testData.materialTypeId },
+            },
+          ],
+        }).then((specialInstanceIds) => {
+          itemData.instanceId = specialInstanceIds.instanceId;
+          itemData.holdingId = specialInstanceIds.holdingIds[0].id;
+          itemData.itemId = specialInstanceIds.holdingIds[0].itemIds;
+        });
+      });
+
+    RequestPolicy.createViaApi(requestPolicyBody);
+    PatronGroups.createViaApi(patronGroup.name).then((res) => {
+      patronGroup.id = res;
+      cy.createTempUser(
+        [
+          permissions.circulationLogAll.gui,
+          permissions.uiCirculationSettingsNoticeTemplates.gui,
+          permissions.uiCirculationSettingsNoticePolicies.gui,
+          permissions.requestsAll.gui,
+        ],
+        patronGroup.name
+      )
+        .then((userProperties) => {
+          userData = userProperties;
+        })
+        .then(() => {
+          UserEdit.addServicePointViaApi(
+            testData.userServicePoint.id,
+            userData.userId,
+            testData.userServicePoint.id
+          );
+
+          cy.getCirculationRules().then((response) => {
+            testData.baseRules = response.rulesAsText;
+            testData.ruleProps = CirculationRules.getRuleProps(response.rulesAsText);
```

Comment:
let's save baseRules right before adding the new rule

Reply:
done